### PR TITLE
Add Marcel

### PR DESCRIPTION
**Pull request by marcelroed**
Sadly this run died after 40 minutes of unknown causes https://api.wandb.ai/links/marcelroed/6he1mqre

The best value was a validation loss of 3.502.

Compile, rotary embeddings, larger model size (and proportionally large FF), tweaking learning rate

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks!
